lib/asciidoctor/latex/converter.rb

```diff
@@ -89,35 +89,47 @@
 module Asciidoctor::LaTeX
   module Html5ConverterExtensions
 
-    def environment node
+    ENV_CSS = "+++<div style='line-height:1.5em;font-size:1.05em;font-style:oblique;margin-bottom:1.5em'>+++"
+    DIV_END = '+++</div>+++'
+    TABLE_ROW_END = '+++</tr></table>+++'
+
+    def info node
+
+      attrs =  node.attributes
+
+      warn "\n    node: #{node.node_name}".cyan
+      warn "   attrs: #{attrs}".cyan
+      warn "   title: #{attrs['title']}".cyan
+      warn "    role: #{attrs['role']}".cyan
+      warn "   level: #{attrs['level']}".cyan
+      warn " options: #{attrs['options']}".cyan
+      warn " type   : #{attrs['type']}".cyan
+      warn " caption: #{node.caption}".red
+      warn "      id: #{attrs['id']}".cyan
+      warn " content: #{node.content}".blue
```

Comment:
I’ve been actually trying to refactor logging, but stopped in middle because from my point of view, most of these log messages seems to be unnecessary even for the debug level.

Therefore, could you please consider which messages are actually useful, mainly for users of the backend, and remove the rest?